android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java

```diff
@@ -1563,6 +1563,9 @@ private void validateAccessToken(String accessToken) {
     @UiThread
     public void setAccessToken(@NonNull String accessToken) {
         // validateAccessToken does the null check
+        if (!TextUtils.isEmpty(accessToken)) {
+            accessToken = accessToken.trim();
```

Comment:
Gotta do a Null check first (which is what TextUtils.isEmpty() is doing).  If not when a null shows up the trim will crash things.